tests/test_linearity.py

```diff
@@ -324,6 +373,11 @@ def _check_linearity_spline(self, do_pd_offsets=False, n_points=200):
             self.input_dims,
         ).outputLinearizer
 
+        if do_weight_fit:
+            # These checks currently fail, and weight fitting is not
```

Comment:
Is there a ticket to investigate/fix the weight fitting that could be added here?